Project.toml

```diff
@@ -7,7 +7,7 @@ BitBasis = "50ba71b6-fa0f-514d-ae9a-0916efc90dcf"
 Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
```

Comment:
just remember to have minor version bump (instead of patch) to be sure on the Graphs update